src/SingleReview.jsx

```diff
@@ -0,0 +1,94 @@
+import {useParams} from 'react-router-dom'
+import {useState,useEffect} from 'react'
+import { getSingleReview } from './api'
+import { formatDate } from './api'
+import AddReviewLikes from './AddReviewLikes'
+
+function SingleReview() {
+    const [singleReview,setSingleReview] = useState({})
+    const {review_id} = useParams();
+    
+
+    const [isLoading, setIsLoading] = useState(false)
+
+    const [finishedLoading, setFinishedLoading] = useState(false)
+    useEffect(() => {
+        setIsLoading(true)
+        setFinishedLoading(false)
+        getSingleReview(review_id).then((review) => {
+            setSingleReview(review)
+            setFinishedLoading(true)
+            const animation = document.querySelector('.rocket')
+            animation.addEventListener('animationend', () => {
+                setIsLoading(false)
+            })
+
+            animation.removeEventListener('animationend', () => {
+                setIsLoading(false)
+            })
+
+
+        })
+        return () => {
+        }
+    }, [])
+    return isLoading ?
+        (<div className="loading-container">
+        <p className="loading-message">Loading...</p>
+        <p className={!finishedLoading ? "rocket loading-rocket" : "rocket rocket-launch"}>
+            <br />
+            🚀
+        </p>
+        </div>)
+        :
+
+    (
+        <div className="single-review">
```

Comment:
Is there a more semantic HTML tag you could use here to avoid the div that describes the content better?